app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:email, :password, :name) }
+    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:password, :current_password) }
+  end
 end
```

Comment:
- [ ] Kindly note that you need to protect your routes so that unauthenticated users cannot access them. Kindy make sure you add the callback that will handle that case `before_action :authenticate_user!`.

### Below here I have attached a video review summary, so that you may understand better, feel free to ask any question if something was not clear

https://drive.google.com/file/d/10uZcEW4M4CiknNu_h6-odl4i0D_oB9vx/view